0.3.0

chrisheninger opened the pull request:
I've gone ahead and added a bunch of stuff to this project based on the work I did with react-drop-n-crop.

I'd like to propose we rename this project in 0.3.0 to `react-google-places` to establish a pattern for our react components. I'm looking forward to maintaining a *well-understood* collection of libraries we can extend across all our apps– with an emphasis on ease of customization/styling.  Exported component would be `<GooglePlaces />` in JSX.

- [x] Storybook demo

- [x] package.json package and script updates

- [x] `prettier` all the things

- [x] Update readme

Commit:
Updates for 0.3.0 bump and proposal for renaming– docs/example WIP

Commit:
More documentation and updates

Commit:
Rename plus more docs

Commit:
Add box-sizing to storybook demo, fix index

agerbens commented on README.md:
````diff
@@ -1 +1,118 @@
+
+#### Using with a ReduxForm (v6) `Field` Component
+```
+// TODO: Add simple example to storybook + readme
````
Should either fill this out or remove the section

agerbens commented on README.md:
```diff
@@ -1 +1,118 @@
+
+#### value: (required)
+
+`onChange` is the callback `function` necessary to update the parent component with the final cropped image file. `onChange` receives a `File` object as an argument.
```
copy/paste artifact

agerbens commented on storybook/stories/SetStateExample.js:
```diff
@@ -0,0 +1,99 @@
+      <div>
+        {!googleApiLoaded
+          ? <Script
+              url="https://maps.googleapis.com/maps/api/js?key=AIzaSyBDvgbceyFAP8QbGpNSVt45PPGDUylFokI&amp;libraries=places"
```
Should require that a user enter their own key as a prerequisite for running this, rather than just hardcoding one in. Keys are easy to get, and you can just have a step of "Get a key and plug it in here" as part of the setup.

Commit:
Clean up component and example, finish readme

Commit:
Not sure why shallow-compare is in this package, we're not using it

chrisheninger commented:
Addressed your points and cleaned up the documentation– just need to figure out how to deal with the Google Places API Key for the github.io demo website as well as development purposes. I think I'm able to http-referrer-restrict where the key is used... we can discuss this in person.

Commit:
Don't rename this project

Commit:
Scripts and package version cleanup

Commit:
Forgot lintstaged file
